object-store-client-common/src/main/scala/uk/gov/hmrc/objectstore/client/ObjectStoreClient.scala

```diff
@@ -33,20 +34,28 @@ class ObjectStoreClient[F[_], REQ_BODY, RES, RES_BODY](
   private val authorizationHeader: (String, String) =
     ("Authorization", config.authorizationToken)
 
+  private def expirationPeriodHeader(expirationPeriod: ObjectExpirationPeriod): (String, String) =
+    "X-EXPIRE-AFTER" -> expirationPeriod.toString
```

Comment:
I don't think the whole thing should be capitalised. So it should be `X-Expire-After`. Though `X-Retention-Period` might make more sense perhaps?